### DISCUSSION TITLE
Collection, bring back different sections

### DISCUSSION THREAD

**Discussion by Matth7878** [trakt/showly#678] [General]
Collection screen has been changed in last version (3.19)
-> now there is only one section with option for only an other one for recently added without any choice for how to display it
-> before there was recently finished, watched and you could choose how you wanted it to appears

Bring back old screen it was perfect. Now not so much... IMHO it's a big regression (unless I didn't see where the settings was) ☹️

**Comment by blackholeearth** [comment]
I definitely agree. 

Make it optional. 

I was using finished section to detect, if an ongoing serie falls into finished section, that means trakt data is not uptodate.
So, I goto trakt and update series data. 

Now it become harder to see , at first glance.

----------
My use case::
looking all status all at once , only makes sense if sorting is recently watched .

1) Watch copule of series.
2)  come back to see if any ongoing series fall into finished section. .
3) If so , update tmdb meta , report to trakt

Thats exactly how i used that section.

Otherwise if sorted by title . They stay constantly , and it doesnt have any added value. Just takes up space.

---------
Tldr;;

So its all about  , how people use it. Some need it, some dont. 
It should be optional. In the setting menu.

**Comment by michaldrabik** [comment]
Converted to Discussion as it's not really an issue.

In general, the horizontal scrolling lists were not very convenient with bigger collections. Majority of users did not like it. 
They were simply moved to filters. You can still access them by tapping "All" and selecting proper section which can be sorted to whataver liking if there is a need.
